arm/Microsoft.Security/azureSecurityCenter/deploy.bicep

```diff
@@ -269,5 +240,11 @@ resource workspaceSettings 'Microsoft.Security/workspaceSettings@2017-08-01-prev
   ]
 }
 
-@description('The resource IDs of the used log analytics workspace.')
+@description('The resource ID of the used log analytics workspace.')
 output workspaceId string = workspaceId
+
+@description('The resource ID of the security center.')
```

Comment:
Should we explain somewhere why the resourceId is empty (we need that to pass the static validation). Still not sure if that is the right approach wrt having an exception in the global tests, which I'm not a huge fan of either

Reply:
Agreed. Alternatively, I was thinking we could return whatever IDs the deployed resources return - but I somehow doubt it would be useful.

How about we provide the description `@description('The resource ID of the security center. Note: This value is empty.')`

I wouldn't call out our Pester tests explicitely - at least not inside the module. 

Last but not least I was thinking we could also adjust the test by
- Fetching from the Folder-Path what the primary resource is
- Check if there is such a resource in the template 
  - if not, like in case of the Security Center, no ID would be required
  - if so, make sure the *resourceID output exists

What do you think?

Reply:
@eriqua I don't really want to block this PR due to this change. Will undo the fix for the security center and we can approach the red pipeline in a different PR

Reply:
Created a new bug in #1473